tailwind.config.ts

```diff
@@ -16,17 +16,16 @@ import plugin from 'tailwindcss/plugin'
 const themeColors = {
     'primary': '#0A3B2F',
     'secondary': '#70C247',
-    'neutral': '#EEEDF0',
     'stone': '#1D1D1D',
-    'amber-900': '#78350F',
     'amber-500': '#F59E0B',
-    'neutral-400': '#999999',
+    'amber-900': '#78350F',
     'neutral-100': '#F7F7F7',
+    'neutral-200': '#F2F2F2',
+    'neutral-400': '#999999',
     'orange-800': '#9A3412',
     'orange-700': '#C2410C',
-    'error': '#FF0000',
```

Comment:
Let's not use built-in color names to declare Tailwin colors otherwise we will loose the whole color palette of the built-in colors and feel hawk-tued